glue/main.py

```diff
@@ -45,17 +49,26 @@ def load_plugins(splash=None, require_qt_plugins=False):
     config = PluginConfig.load()
 
     n_plugins = len(list(iter_plugin_entry_points()))
```

Comment:
I think it might be best to actually define ``n_plugins`` as ``len(plugins_to_use)`` once ``plugins_to_use`` has been defined.